main.go

```diff
@@ -91,6 +125,53 @@ func cleanupImages(ecrCli *ecr.ECR, repoName string, images []*ecr.ImageDetail,
 	return nil
 }
 
+func excludeImages(srcImages []*ecr.ImageDetail, excludeImages []*ecr.ImageDetail) (result []*ecr.ImageDetail) {
+	var addImg bool
+	result = srcImages[:0]
+
+	for _, image := range srcImages {
+		addImg = true
+		for _, excImage := range excludeImages {
+			if image.ImageDigest == excImage.ImageDigest {
+				addImg = false
+				break
```

Comment:
You can simplify this pattern by just adding the image to result here.

Reply:
I can't in this one I'm excluding slice from another slice. pretty much I'm removing items if they are present in another slice.

I'll try to modify filterTags so it would return 2 slices with and without tags matched by regexp.